robolectric-resources/src/main/java/org/robolectric/res/DocumentLoader.java

```diff
@@ -43,8 +43,9 @@ private void loadFile(FsFile dir, XmlLoader[] xmlLoaders) {
 
   private void loadResourceXmlFile(FsFile fsFile, XmlLoader... xmlLoaders) {
     VTDNav vtdNav = parse(fsFile);
+    XmlContext xmlContext = new XmlContext(packageName, fsFile);
     for (XmlLoader xmlLoader : xmlLoaders) {
-      xmlLoader.processResourceXml(fsFile, vtdNav, packageName);
+      xmlLoader.processResourceXml(new XpathResourceXmlLoader.XmlNode(vtdNav), xmlContext);
```

Comment:
Am I totally wrong here or can `new XpathResourceXmlLoader.XmlNode(vtdNav)` happen above the `for` loop, too?